src/UnitTests.BinSkim.Rules/UnitTests.BinSkim.Rules.csproj

```diff
@@ -0,0 +1,125 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this be BinSkim.Rules.UnitTests (to match the naming convention for the FunctionalTests projects?)